src/cli/cli_br.cpp

```diff
@@ -44,6 +44,32 @@
 namespace ot {
 namespace Cli {
 
+/**
+ * @cli br init
+ * @code
+ * br init 2 1
+ * Done
+ * @endcode
+ * @cparam br init @ca{if-index} @ca{is-running}
```

Comment:
Can we elaborate a little on what the if-index parameter is?  Or maybe spell it out interface-index

For myself, I have no clue where that comes from (how to identify the index).  The API reference doesn't explain it either.

Reply:
Updated it to use `infrastructure-interface-index`.

This is the netif index for the adjacent infrastructure link, which is the infra netif on which we act as a Border Router (BR).

In my view, this API would have been better as a platform API rather than a public OpenThread API. Defining it in `platform/infra_if.h` would have been a better fit, as it already matches the existing `otPlatInfraIfStateChanged()` which is used to notify when the infra netif state changes. However, we generally try to avoid changing public APIs (to avoid compatibility issues), so it is tricky to change this now.

Ideally, an OT user should not need to use the `otBorderRoutingInit()` API. It should be called by the platform layer. I added it to the CLI in this PR only to be able to use it in the test-case script.

Reply:
Sounds good, thanks for the explanation.